frontend/src/components/layouts/AppLayout.tsx

```diff
@@ -295,8 +295,8 @@ export default function AppLayout() {
                 >
                   <nav className="grid gap-2 text-lg font-medium">
                     <div className="p-3 ">
-                      <Link to="/" className="font-semibold text-xl">
-                        <span className="">Alby Hub</span>
+                      <Link to="/">
+                        <AlbyHubLogo className="text-foreground" />
```

Comment:
Why text-foreground? That should be the default for text, right?

Reply:
I saw this being done similarly above, in [L220](https://github.com/getAlby/hub/blob/master/frontend/src/components/layouts/AppLayout.tsx#L220)

Reply:
We did this so that the theme primary color does not apply. Sadly this is only the case for the yellow theme on light mode where it doesn't look good :(